planetiler-core/src/main/java/com/onthegomap/planetiler/util/Wikidata.java

```diff
@@ -312,13 +317,19 @@ private LongObjectMap<Map<String, String>> queryWikidata(List<Long> qidsToFetch)
           LOGGER.error("sparql query failed, exhausted retries: " + e);
           throw e;
         }
+        if (e.getMessage() != null && e.getMessage().contains("GOAWAY")) {
+          try {
+            LOGGER.debug("GOAWAY received, waiting {} ms to give server some room", config.httpRetryWait());
+            Thread.sleep(config.httpRetryWait());
+          } catch (InterruptedException e2) {
+            Thread.currentThread().interrupt();
+          }
+        }
```

Comment:
Can we just do the wait in all cases, in case there's another kind of "please back off" response that looks slightly different?

Also this method can throw `InterruptedException` is there a reason we wouldn't want to let that propagate?